alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultProviderDataAccessor.java

```diff
@@ -1,18 +1,18 @@
 /**
  * alert-database
- * <p>
```

Comment:
Does someone have the wrong version of our save actions? I've seen this a bunch recently.

Reply:
I too have seen this a lot recently. Not sure why it changes a few files each time.

Reply:
The copyright should be coming from the common-gradle-plugin when you run a build.  Should not be from the save actions as far as I knew.